src/features/cookies/helpers/useIsCookiesListUpToDate.native.test.ts

```diff
@@ -0,0 +1,121 @@
+import mockdate from 'mockdate'
+
+import { useIsCookiesListUpToDate } from 'features/cookies/helpers/useIsCookiesListUpToDate'
+import * as Firestore from 'libs/firebase/firestore/getCookiesLastUpdate'
+import { storage } from 'libs/storage'
+import { flushAllPromisesWithAct, renderHook } from 'tests/utils'
+
+import Package from '../../../../package.json'
+
+const DAY_IN_MS = 24 * 60 * 60 * 1000
+const COOKIES_CONSENT_KEY = 'cookies_consent'
+const today = new Date('2022-09-16')
+const tomorrow = new Date(today.getTime() + DAY_IN_MS)
+const aSecondAgo = new Date(today.getTime() - 1000)
+const lastWeek = new Date(today.getTime() - 7 * DAY_IN_MS)
+const threeMonthsAgo = new Date(today.getTime() - 180 * DAY_IN_MS)
+const mockToday = today
+mockdate.set(today)
+
+const mockFirebaseData: {
+  lastUpdated: Date
+  lastUpdateBuildVersion: number
+} = { lastUpdated: mockToday, lastUpdateBuildVersion: Package.build }
+
+jest.spyOn(Firestore, 'getCookiesLastUpdate').mockResolvedValue(mockFirebaseData)
```

Comment:
plutot que de faire un mock et un objet séparés, avec des effets de bords

je conseille de faire seulement un mock

```suggestion
const defaultMockFirebase: CookiesLastUpdate = {
  lastUpdated: mockToday,
  lastUpdateBuildVersion: Package.build,
}
const mockFirebase = jest.spyOn(Firestore, 'getCookiesLastUpdate').mockResolvedValue(defaultMockFirebase)
```

avec des `mockResolvedValueOnce` dans les tests qui ont besoin de changer la valeur

```ts
mockFirebase.mockResolvedValueOnce({
  ...defaultMockFirebase,
  lastUpdateBuildVersion: Package.build + 1,
})
```

Reply:
Oh c'est vachement plus propre merci ! 🙌